### PR TITLE
Bumped version of Graph API to 2.10

### PR DESCRIPTION
**Pull request by piotrjaworski**
Facebook Graph API v2.6 will be deprecated soon, I bumped version to v2.10. 

Basically, there is no need to do any code changes because of that.

### CODE REVIEW

**Comment by piotrjaworski** [comment]
@mkdynamic please check the following changes, thanks!

**Comment by simi** [comment]
Hello! Thanks for pull request. If you want to help me merging this, can you summarize difference between 2.6 and 2.10? That will be great to decide if we need to bump minor or patch level of version.

**Comment by piotrjaworski** [comment]
@simi sure!

2.7 - https://developers.facebook.com/docs/graph-api/changelog/version2.7#mapi
2.8 - https://developers.facebook.com/docs/graph-api/changelog/version2.8#mapi
2.9 - https://developers.facebook.com/docs/graph-api/changelog/version2.9#mapi
2.10 - https://developers.facebook.com/docs/graph-api/changelog/version2.10#mapi

Basically, there are some changes/new features but nothing touches OAuth - as far as I checked.

**Comment by piotrjaworski** [comment]
@simi @mkdynamic Is there any possibility to merge it?

**Comment by simi** [comment]
@piotrjaworski I'm ok with this change with releasing new major version (5.0.0).

**Comment by piotrjaworski** [comment]
@simi I tagged it as 5.0.0 - it's ready to merge!

**Comment by simi** [comment]
@piotrjaworski can you please do last check before release? Do you think everything is prepared and ready for release?

**Comment by piotrjaworski** [comment]
@simi It has already been tested on a big production app - everything works well!
